### PR TITLE
Dont throw exception but display unknown mission

### PR DESCRIPTION
**Pull request by 1-alex98**
Prepares #1134

### CODE REVIEW

File: src/main/resources/i18n/messages.properties [hunk 1]
```diff
@@ -50,6 +50,7 @@ play.coop=Co-Op
 
 leaderboard.failedToLoad=Leaderboard could not be loaded
 leaderboard.loading=Loading leaderboard…
+coop.unknownMission=unknown mission
```
Comment (Brutus5000) [inline src/main/resources/i18n/messages.properties]: I'd start with an uppercase letter like in 99% of all other messages.

File: src/main/java/com/faforever/client/coop/CoopController.java [hunk 2]
```diff
@@ -160,17 +163,22 @@ public void initialize() {
...
       Platform.runLater(() -> {
         missionComboBox.setItems(observableList(coopMaps));
         GamesTableController gamesTableController = uiService.loadFxml("theme/play/games_table.fxml");
-        Function<String, String> coopMissionNameProvider = mapFolderName -> coopMaps.stream()
-            .filter(coopMission -> coopMission.getMapFolderName().equalsIgnoreCase(mapFolderName))
-            .findFirst()
-            .orElseThrow(() -> new IllegalStateException("Map folder name did not match with any coop mission"))
-            .getName();
+
+        Function<String, String> coopMissionNameProvider = (mapFolderName -> {
```
Comment (Brutus5000) [inline src/main/java/com/faforever/client/coop/CoopController.java]: It's too big now. We should put it into a new function imho.

**Review by Brutus5000 — CHANGES_REQUESTED** [review]

**Comment by 1-alex98** [comment]
Please somebody review

**Comment by 1-alex98** [comment]
@Geosearchef would u mind reviewing?

**Review by micheljung — APPROVED** [review]

**Pull request merged**